iocore/net/quic/QUICTypes.h

```diff
@@ -424,13 +424,20 @@ class QUICPreferredAddress
   bool _valid = false;
 };
 
-enum class QUICStreamType : uint8_t {
+enum class QUICStreamInitType : uint8_t {
   CLIENT_BIDI = 0x00,
```

Comment:
Not sure what name should be used here?

Reply:
I'd keep this as is and name the other QUICStreamDirection.